crates/subspace-node/Cargo.toml

```diff
@@ -54,7 +54,7 @@ sp-core = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrat
 sp-runtime = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", rev = "b6c1c1bcfa5d831bfd1f278064d7af757f9b38f5" }
 sp-timestamp = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", rev = "b6c1c1bcfa5d831bfd1f278064d7af757f9b38f5" }
 structopt = "0.3.22"
-subspace-runtime = { version = "0.1.0", path = "../subspace-runtime" }
+subspace-runtime = { version = "0.1.0", features = ["do-not-enforce-cost-of-storage"], path = "../subspace-runtime" }
```

Comment:
So it's not enabled at the moment, just a question, when/in which cases it will be used?

Reply:
That is to be determined. The issue is that cost of storage with initial parameters is ~8.2 SSC/byte and goes way higher once we have some history and parameters are recomputed. I don't have a clear answer here and it is not sufficient to have certain number of blocks as a special case. I guess we'll have to discuss roll-out plan separately on a call tomorrow.

Reply:
Yes